docs/hyn/5.4/configuration.md

```diff
@@ -18,12 +18,13 @@ I've listed some of the basic settings below.
 
 ## tenancy.php
 
-The tenancy configuration file holds every setting related to tenants. This includes how to generate a unique
-website, but also how database connections are set up. In addition, it allows for;
+The `tenancy.php` configuration file holds every setting related to tenants. 
+This includes how to generate a unique website, but also how database connections are set up. 
+In addition, it allows for:
```

Comment:
Newlining these three sentences won't work with markdown. Sentences in markdown have to be separated by an empty (new)line.

Reply:
Agreed. I just did it in the markdown for easier readability for whomever is maintaining the doc.